e2e/journey-app/server-configs.ts

```diff
@@ -13,4 +13,34 @@ export const serverConfigs: Record<string, JourneyClientConfig> = {
     },
     realmPath: '/alpha',
   },
+  ['TEST_WebAuthn-Registration']: {
```

Comment:
We can think about how to actually do this if the objects the same every time.

Reply:
Yeah, I wasn't sure exactly how we wanted to handle different configs. I ended up not using it at all in all the tests I moved over from the legacy repo.